zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java

```diff
@@ -157,11 +157,15 @@ public static Collection getRealmsList() {
             );
             if (auth != null) {
               roles = new HashSet<>(auth.getRoles());
+              break;
+            }
+          } catch (Throwable t) {
+            if (log.isDebugEnabled()) {
+              String msg = "Realm [" + realm +
+                           "] threw an exception during a multi-realm authorization attempt:";
+              log.debug(msg, t);
```

Comment:
Is it OK for swallow the error via `log.debug` ? Would it be better to use `log.warn` ?

Reply:
Log.warn is fine I guess.  I am hoping to  make similar changes to the master branch this week.